src/odemis/acq/fastem.py

```diff
@@ -962,49 +963,60 @@ def estimateTiledAcquisitionTime(stream, stage, area):
 
     return acq_time_overview
 
+class OverviewAcquisition(object):
+    """Class to run the acquisition of one overview image (typically one scintillator)."""
+    def __init__(self) -> None:
+        self.sf = None
 
-def _run_overview_acquisition(f, stream, stage, area, live_stream):
-    """
-    Runs the acquisition of one overview image (typically one scintillator).
+    def _cancel_acquisition(self, future):
+        if self.sf:
+            self.sf.cancel()
 
-    :param f: (ProgressiveFuture) Acquisition future object.
-    :param stream: (SEMstream) The stream used for the acquisition.
-    :param stage: (actuator.MultiplexActuator) The stage in the sample carrier coordinate system.
-        The x and y axes are aligned with the x and y axes of the ebeam scanner.
-    :param area: (float, float, float, float) xmin, ymin, xmax, ymax coordinates of the overview region.
-    :param live_stream: (StaticStream or None): StaticStream to be updated with each tile acquired,
-           to build up live the whole acquisition. NOT SUPPORTED YET.
+    def run(self, f, stream, stage, area, live_stream):
```

Comment:
Instead of getting the future as argument here, I'd suggest to get it in the `__init__()`. For such "Task" class, which represent one asynchronous execution, it's the common way to do it, and I feel it's more logical.